api-reference/overview.mdx

```diff
@@ -0,0 +1,27 @@
+
+<Note>It's important to note that our API and backend SDK are intended for use exclusively in server-side applications.
+**Attempting to use them in a client-side application will result in Cross-Origin Resource Sharing (CORS) errors.**
+This restriction ensures the security and integrity of our services.</Note>
+
+
+## Authentication
+
+Authentication for the Novu API involves the use of an API Key, which is a secure credential that is tied to your Novu account. This key should be included in the header of the request in the Authorization field as a string prefixed with 'ApiKey '.
+
+For example, when using Novu in a Node.js application, the Novu package should be imported and initialized with the API key, as shown in this snippet:
+
+```javascript
+import { Novu } from '@novu/node';
+const novu = new Novu('<NOVU_API_KEY>');
+```
+
+Replace `<NOVU_API_KEY>` with your actual API Key, available in the API Key section of the Novu Dashboard. 
+<Warning> It is advised not to hardcode your credentials in a file in production environments. Use environment variables instead. </Warning>
+
+## API Endpoints
+Novu provides a multitude of API endpoints that enable a variety of functionalities.
+the base URL for the Novu API is `https://api.novu.co/v1`. 
+
```

Comment:
mention we have US and EU API. docs API reference by default uses US API:- https://api.novu.co/v1
EU version is:-  https://eu.api.novu.co/v1

Reply:
True!